src/workerd/api/crypto/dh.c++

```diff
@@ -100,6 +103,10 @@ kj::Own<DH> initDh(kj::OneOf<kj::Array<kj::byte>, int>& sizeOrKey,
             }
             return 1;
           };
+          // Operations on an "egregiously large" prime will throw with recent boringssl.
+          if (size > OPENSSL_DH_MAX_MODULUS_BITS) {
+            KJ_LOG(WARNING, "DiffieHellman init: requested prime size too large");
```

Comment:
Should these be `NOSENTRY` or do we want them appearing there?

Reply:
I do prefer having warnings appear on Sentry as it makes them easier to find. These should be temporary so hopefully this is fine